browser/android/preferences/brave_pref_service_bridge.cc

```diff
@@ -260,6 +260,13 @@ void JNI_BravePrefServiceBridge_SetReferralCheckedForPromoCodeFile(
       kReferralCheckedForPromoCodeFile, value);
 }
 
+void JNI_BravePrefServiceBridge_SetReferralReportedInstall(
```

Comment:
Should this be called somewhere?

Reply:
I think we don't need to add this because `kReferralReportedInstall` is newly added prefs.
The only case that set promo related flags from java layer is for migrating existing configs. cc @samartnik

Reply:
Yep, good point. Can just be removed, most likely.